include/proto.h

```diff
@@ -595,6 +595,9 @@ ChessInfo* reversi_replay(FILE* fp);
 /* screen/pfterm (ncurses-like) */
 void initscr	(void);
 int  resizeterm	(int rows, int cols);
+#ifdef USE_PFTERM
+int  resizeterm_true(int rows, int cols, int rows_true, int cols_true);
```

Comment:
Please add the API for both implementations. We do not want to special case for pfterm; instead switch to the new API for both. I'd keep the original resizeterm(). For `screen.c` you can just pass through to original `resizeterm()`.

I'd call the new function something like `resizeterm_with_win()`. I'd have the first two parameters as the old API, and add the "limited window" size as the latter two new parameters.

Reply:
Sure. As for the new API, I had not come up with a better name for the new function. The parameters have already had the suggested semantics.

The function signature will be changed into:
```c
int resizeterm_with_win(int rows, int cols, int rows_win, int cols_win);
```